WordPress/src/main/java/org/wordpress/android/util/config/JetpackPoweredFeatureConfig.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.FeatureInDevelopment
+import javax.inject.Inject
+
+/**
+ * Configuration for Jetpack Powered indicators
+ *
+ * TODO: When it is ready to be rolled out uncomment the lines 12 and 19, remove line 13 and this to-do
+ */
+// @Feature(JetpackPoweredFeatureConfig.JETPACK_POWERED_REMOTE_FIELD, true)
+@FeatureInDevelopment
+class JetpackPoweredFeatureConfig @Inject constructor(
+    appConfig: AppConfig
+) : FeatureConfig(
+        appConfig,
+        BuildConfig.JETPACK_POWERED,
```

Comment:
This is very nice, a practical confirmation that after the `Ktlint` [removal](https://github.com/wordpress-mobile/WordPress-Android/pull/16828), we can use trailing commas with no issues, as the CI checks are all ✅.